app/views/users/show.html.erb

```diff
@@ -20,6 +20,6 @@
       </li>
     <%end%>
   </ul>
-  <button><%= link_to "See all posts", user_recent_posts_path(@author)%></button>
+  <button><%= link_to "See all posts", user_post_index_path(@author)%></button>
   <button><%= link_to 'Create new post', new_post_path(@user)%></button>
```

Comment:
- [ ] It would be nice if you add authorization rules for creating a post as well because I can see create new post button on all the users suppose I am signed In with user 1 then I can also see the create button in the user 2 profile :fearful: Kindly fix this issue as well because this way your authorization would not be implemented properly :thinking:


- [ ] Kindly fix all your paths with all the buttons because I had to fix a lot of paths :fearful: and I was not able to review it easily :relaxed: Kindly fix all the paths as well so the next reviewer won't face any issue :+1:

**[OPTIONAL]**

- [ ] It would be nice if you add a letter-opener gem in your application because once a user is created then it asks for confirmation so in this regards letter opener will be a blessing :+1: